src/consensus/pbft/libbyz/Replica.cpp

```diff
@@ -343,31 +334,104 @@ bool Replica::compare_execution_results(
              << ", info.ctx:" << info.ctx << std::endl;
     return false;
   }
+
+  if (!std::equal(
```

Comment:
Any particular reason this got moved down?

Reply:
Yes we are probably going to remove the full state merkle tree so I was focused on the replicated one so it got moved up so it can be compared before the full state one fails. Doesn't really make a difference, was done mostly for debugging